buildqtc.sh

```diff
@@ -408,6 +408,13 @@ build_windows_qtc() {
 	local opengl32sw_lib="opengl32sw-32.7z"
 	curl -s -f -o $opengl32sw_lib http://$OPT_UPLOAD_HOST/sailfishos/win32-binary-artifacts/opengl/$opengl32sw_lib
 
+    local pkg_config_libs="pkg-config_0.26-1_win32.zip \
+        gettext-runtime_0.18.1.1-2_win32.zip \
+        glib_2.28.8-1_win32.zip"
```

Comment:
Where do these come from? Is it documented somewhere?

Reply:
will be documented in sdk-docs

Reply:
but we have the README as well - I will update it in a sec